ledger/tracker.go

```diff
@@ -333,8 +336,22 @@ func (tr *trackerRegistry) initialize(l ledgerForTracker, trackers []ledgerTrack
 }
 
 func (tr *trackerRegistry) loadFromDisk(l ledgerForTracker) error {
+	var dbRound basics.Round
+	err := tr.dbs.Snapshot(func(ctx context.Context, tx trackerdb.SnapshotScope) (err error) {
+		ar, err := tx.MakeAccountsReader()
+		if err != nil {
+			return err
+		}
+
+		dbRound, err = ar.AccountsRound()
+		return err
+	})
+	if err != nil {
+		return err
+	}
```

Comment:
Just curious - Why did this get moved from `initialize`?

Reply:
the main motivation was to simplify testing `trackerRegistry.initialize`. Without this change need to call db init procedures. Plus, reading db round from disk appears to belong to `loadFromDisk`.